src/components/WorkOrder/AddImage.js

```diff
@@ -90,13 +90,13 @@ class AddImage extends Component {
     const buttons = this.state.captured ? (
       <div className={pictureButtonStyles}>
         <button
-          className="deleteButton btn btn-danger"
+          className="deleteButton btn btn-danger btn-lg"
```

Comment:
Just wondering, why not use the new `<Button/>` component here too?

Reply:
@mateoclarke I started having the same question about `<Button />`. I started thinking this component might make more sense as `<LinkButton />` or maybe disabling the `<Link></Link>` wrapper if no link is passed. 🤔 What do you think? I'll merge for now so we can discuss where to go with this.